opendm/video/srtparser.py

```diff
@@ -0,0 +1,124 @@
+from datetime import datetime
+import re
+class SrtFileParser:
+    def __init__(self, filename, utc_offset):
+        self.filename = filename
+        self.data = []
+        self.utc_offset = utc_offset
+
+    def get_entry(self, timestamp):
+        if not self.data:
+            self.parse()
+
+        for entry in self.data:
+            if entry["timestamp"] <= timestamp:
```

Comment:
Assuming self.data is sorted from least to most recent, which I think it is by reading `parse`, doesn't this end up returning always the first frame?

Can you link the test data you're using?